packages/mobile/src/screens/verify-phone-screens/verify-code/verify-code.tsx

```diff
@@ -0,0 +1,160 @@
+import React, { FC } from 'react';
+import { PhoneVerifyDto } from '@vse-bude/shared';
+import {
+  useAppDispatch,
+  useAppForm,
+  useAppSelector,
+  useCustomTheme,
+  useNavigation,
+  useTranslation,
+  useState,
+} from '~/hooks/hooks';
+import {
+  ButtonAppearance,
+  DataStatus,
+  RootScreenName,
+} from '~/common/enums/enums';
+import {
+  CheckCircleIcon,
+  Input,
+  KeyboardAvoiding,
+  PrimaryButton,
+  View,
+} from '~/components/components';
+import { verifyPhoneActions } from '~/store/actions';
+import { images } from '~/assets/images/images';
+import { globalStyles } from '~/styles/styles';
+import { RootNavigationProps } from '~/common/types/types';
+import {
+  selectAuthDataStatus,
+  selectVerifyPhoneDataStatus,
+  selectUserPhone,
+} from '~/store/selectors';
+import { notification } from '~/services/services';
+import { CODE_REGEX } from '~/common/regexp/regexp';
+import {
+  ButtonsContainer,
+  CustomText,
+  Header,
+  Title,
+  VerifyImage,
+  Wrapper,
+} from '../components/components';
+import { styles } from './styles';
+
+const VerifyCodeScreen: FC = () => {
+  const { t } = useTranslation();
+  const dispatch = useAppDispatch();
+  const navigation = useNavigation<RootNavigationProps>();
+  const { colors } = useCustomTheme();
+  const userPhone = useAppSelector(selectUserPhone);
+  const dataStatusVerify = useAppSelector(selectVerifyPhoneDataStatus);
+  const dataStatusAuth = useAppSelector(selectAuthDataStatus);
+  const [isValidCode, setIsValidCode] = useState(false);
+  const isLoading =
+    dataStatusVerify === DataStatus.PENDING ||
+    dataStatusAuth === DataStatus.PENDING;
+  const { control, errors, handleSubmit } = useAppForm<PhoneVerifyDto>({
+    defaultValues: {
+      code: '',
+    },
+  });
+
+  const handleBackButtonPress = (): void => {
+    navigation.goBack();
+  };
+
+  const handleResendPress = (): void => {
+    dispatch(verifyPhoneActions.sendCodeVerifyPhone())
+      .unwrap()
+      .then(() => {
+        notification.success(t('verificationPhone.CODE_SENT'));
+      })
+      .catch((err) => {
+        throw err;
```

Comment:
Same here, unless you introduce some sort of error boundary
```suggestion
        console.warn(err);
```

Reply:
corrected